core/lib/basic_types/src/lib.rs

```diff
@@ -139,15 +212,78 @@ impl Default for L1BlockNumber {
     }
 }
 
-impl Default for L2ChainId {
-    fn default() -> Self {
-        Self(270)
```

Comment:
Why did you remove the default?

Reply:
take a look on like 131

Reply:
```
#[allow(clippy::derivable_impls)]
impl Default for L1BatchNumber {
    fn default() -> Self {
        Self(0)
    }
}

``` 

What?

Reply:
<img width="879" alt="image" src="https://github.com/matter-labs/zksync-era/assets/72273339/1db2ba5a-444a-49af-8718-9c74e4fa3cca">